src/components/Avatar/Avatar.js

```diff
@@ -19,11 +19,11 @@ const AVATAR_SIZES_MEDIUM = '60px';
 const AVATAR_SIZES_LARGE = '96px';
```

Comment:
Is the 80x80 avatar image variant large enough for large avatar component which seems to be 96x96? Earlier it has used the 240x240 variant

Reply:
@OtterleyW good question. 96px avatar is used at least on listing page, where it is actually author's avatar and therefore fetched separately. If that's the only place, then it doesn't affect currentUser's avatar sizes.

Reply:
Hmm. TopbarMobileMenu uses it too -> big retina version should be left there since a high pixel density is what you expect in mobile phones. (240px aka square-small should be fine for AvatarLarge component)

Reply:
True, would it make sense to have the variants 40x40, 100x100, and 240x240 (square-small)?

Reply:
There are two parallel threads that discuss the same issue. My suggestion is that you just add 40x40 and 80x80 and keep the current ones there too ('variants.square-small', 'variants.square-small2x') because ProfileSettingsPage needs them.